config/checkstyle_non_main_files_checks.xml

```diff
@@ -24,6 +24,16 @@
     <message key="regexp.filename.mismatch"
       value="CI folder should only contain executables files"/>
   </module>
+  <module name="RegexpOnFilename">
+    <property name="id" value="regexpOnScriptsAndConfigs"/>
+    <property name="folderPattern" value="[\\/].github[\\/]workflows([\\/]|$)"/>
+    <property name="fileNamePattern" value="^([a-z][a-z0-9]*)(-[a-z0-9]+)*\..*$"/>
+    <property name="match" value="false"/>
+    <property name="fileExtensions" value="sh, groovy, cmd, bat, yml"/>
+    <message key="regexp.filename.mismatch"
+             value="All script and configuration files
+             in the ''.github/workflows'' folder should be kebab-case."/>
```

Comment:
Violation will be for single file, let's make it compact  , something like `filename should be kebab-case`

Reply:
Done.